master/buildbot/steps/trigger.py

```diff
@@ -330,4 +343,12 @@ def getResultSummary(self):
     def getCurrentSummary(self):
         if not self.triggeredNames:
             return {'step': 'running'}
-        return {'step': 'triggered {}'.format(', '.join(self.triggeredNames))}
+        summary = ""
+        log.err("trigger current summary %s" % str(self.reslist))
```

Comment:
Debug code?

Reply:
Yes, sorry. Been staring at it for too long now!